test/test_ops.py

```diff
@@ -655,7 +695,7 @@ def test_box_area(self):
         calc_area = ops.box_area(box_tensor)
         assert calc_area.size() == torch.Size([2])
         assert calc_area.dtype == box_tensor.dtype
-        assert torch.all(torch.eq(calc_area, expected)).item() is True
```

Comment:
Lets keep the scope of this PR narrow and fix this in a follow up PR. The same applies to the other changes not related to box conversion.

Reply:
Ok let me undo this and fix in next one 👍

Reply:
Reverted assertion.